lib/citizens_advice_form_builder/form_builder.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "action_view"
+
+module CitizensAdviceFormBuilder
+  class FormBuilder < ActionView::Helpers::FormBuilder
+    def cads_text_field(attribute, label: nil)
+      label_text = label
+
+      @template.content_tag(:fieldset) do
+        label(attribute, label_text) + text_field(attribute)
+      end
+    end
+
+    def cads_button(value = "Save changes")
```

Comment:
I'm interested to see how this form element will evolve when we add the design system. The previous and next buttons look visually different so we will likely end up having different options for buttons (i.e primary, secondary with the different icons)?